src/wiki/development/development-builds.md

```diff
@@ -5,11 +5,12 @@ eleventyNavigation:
   order: 1
 ---
 # Development Builds
+Please understand that **these builds are not intended for most users**. There may be bugs, and other instabilities. You have been warned.
 
-There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). These have debug information in the binaries, so their file sizes are relatively larger.
-Portable builds are provided for AppImage on Linux, Windows, and macOS.
+There are per-commit development builds available [here](https://github.com/PolyMC/PolyMC/actions). Debug information is present within the binaries, making their file sizes larger than otherwise normal.
+Portable dev-builds are, as of now, only provided for Windows.
```

Comment:
```suggestion
Portable dev-builds are provided for Windows and Linux, while there's also a binary Linux tarball, an appimage and a MacOS .app
```

Reply:
@Regular-Baf apply that

Reply:
for some reason it won't let me, the commit button is greyed out.
I'll type it in